FEEDBACK.md

```diff
@@ -24,14 +24,37 @@ Additional Reviews:
 
 
 ## Programs
+### Los Alamos Quantum Computing Summer School (ISTI)
```

Comment:
This already exists in the repo, so this is creating a merge conflict. It looks like your fork is behind my master branch a bit. Either catch up your branch, then re-add the summer school, or you can just re-fork my repo and re-add the summer school